http.c

```diff
@@ -4083,7 +4161,7 @@ evhttp_associate_new_request_with_connection(struct evhttp_connection *evcon)
 	if ((req = evhttp_request_new(evhttp_handle_request, http)) == NULL)
 		return (-1);
 
-	if ((req->remote_host = mm_strdup(evcon->address)) == NULL) {
+	if ((req->remote_host = mm_strdup(evcon->address ?: "localhost")) == NULL) {
```

Comment:
AFAIR visualstudio do not like this syntax (sadly, but, true, 2a4bf294704034f6a7a962931d3028fe6c0b0ac8)

Reply:
And what the purpose of the default to localhost? Some issues in code for unixsocket, but this will hide some possible issues, no? (since you silently fallbackend to localhost)

Reply:
removed